hack/create.sh

```diff
@@ -51,23 +51,8 @@ export RESOURCEGROUP=$1
 rm -rf _data
 mkdir -p _data/_out
 
-# if AZURE_CLIENT_ID is used as AZURE_AAD_CLIENT_ID, script will reset global team account!
-set +x
```

Comment:
Can you also remove this from `test/e2e.sh`?

Reply:
oops, I missed that, thanks.